packages/react-html/src/components/tests/HtmlUpdater.test.tsx

```diff
@@ -142,4 +142,38 @@ describe('<Provider />', () => {
     expect(metaElement).toBe(allMetas[0]);
     expect(allMetas[1].getAttribute('content')).toBe(metaTwo.content);
   });
+
+  it('only keeps the last added meta based on name or property', () => {
```

Comment:
In my previous PR I forgot to test that part... if you run this test in master it will fail.